packages/api/src/controllers/taskController.js

```diff
@@ -679,7 +679,7 @@ const taskController = {
         .whereNotDeleted()
         .withGraphFetched(
           `[locations.[location_defaults], managementPlans, soil_amendment_task, field_work_task.[field_work_task_type], cleaning_task, pest_control_task, 
-            harvest_task.[harvest_use], plant_task, transplant_task, irrigation_task.[irrigation_type]]
+            harvest_task.[harvest_use], plant_task.[planting_management_plan.[bed_method, container_method, broadcast_method, row_method]], transplant_task, irrigation_task.[irrigation_type]]
```

Comment:
This change doesn't actually impact on this specific bug.

However adding this part of the graph was based on `getPlantingTasksAndPlantingManagementPlansSuccessSaga` being set up to get and then forward the `planting_management_property` (which had previously always just not existed). There may be another task + plans bug that is therefore addressed by this; at the least it shouldn't hurt as it's not difficult to fetch this extra part of the graph.